ksrates/fc_configfile.py

```diff
@@ -126,19 +126,44 @@ def get_newick_tree(self):
             logging.error("Unrecognized format for parameter newick_tree in configuration file (for example, parentheses do not match)")
             sys.exit(1)
 
+    def check_complete_dictionary(self, dictionary):
```

Comment:
If this function is specific to `latin_names` name it that way. If not document it in a more general way without referring to `latin_names`